portfolio-project-matching-app/Firebase/clientApp.ts

```diff
@@ -1,7 +1,25 @@
 // Import the functions you need from the SDKs you need
 import { initializeApp } from "firebase/app";
 // TODO: Add SDKs for Firebase products that you want to use
-import { getFirestore, collection, query, where, doc, getDoc, getDocs, addDoc, setDoc } from 'firebase/firestore';
+import {
+  // REFERENCE GETTERS
```

Comment:
Did my best to organize my imports and exports. There are a ton and things can easily get messy.